src/test/operators/join_semi_anti_test.cpp

```diff
@@ -73,14 +73,84 @@ TEST_F(JoinSemiAntiTest, AntiJoinRefSegments) {
   auto scan_b = this->create_table_scan(_table_wrapper_a, ColumnID{0}, PredicateCondition::GreaterThanEquals, 0);
   scan_b->execute();
 
-  test_join_output<JoinHash>(scan_a, scan_b, {ColumnID{0}, ColumnID{0}}, PredicateCondition::Equals, JoinMode::Anti,
-                             "resources/test_data/tbl/joinoperators/anti_int4.tbl", 1);
+  test_join_output<JoinHash>(scan_a, scan_b, {ColumnID{0}, ColumnID{0}}, PredicateCondition::Equals,
+                             JoinMode::AntiDiscardNulls, "resources/test_data/tbl/join_operators/anti_int4.tbl", 1);
 }
 
 TEST_F(JoinSemiAntiTest, AntiJoinBig) {
   test_join_output<JoinHash>(_table_wrapper_semi_a, _table_wrapper_semi_b, {ColumnID{0}, ColumnID{0}},
-                             PredicateCondition::Equals, JoinMode::Anti,
-                             "resources/test_data/tbl/joinoperators/anti_result.tbl", 1);
+                             PredicateCondition::Equals, JoinMode::AntiDiscardNulls,
+                             "resources/test_data/tbl/join_operators/anti_result.tbl", 1);
+}
+
+TEST_F(JoinSemiAntiTest, MultiPredicateSemiJoin) {
```

Comment:
With `JoinMultiPredicateTest`, do these have any purpose? If not, I'd remove them, too many tests isn't necessary